code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -11,6 +11,8 @@
 	slot_flags = SLOT_BELT
 	origin_tech = "bluespace=4;materials=5"
 
+	/// The body/brain of the player inside this construct, transferred over from the soulstone.
+	var/atom/movable/held_body
```

Comment:
this needs to be properly garbage collected with signals

Reply:
I believe its done? Please double check if its fine for you

Reply:
Signals look good! However we still want to null `held_body` in `Destroy` in case the stone is destroyed while it has a ref.

Reply:
Got it